src/app/item-popup/ItemStats.tsx

```diff
@@ -99,6 +98,8 @@ function ItemStatRow({
     }
   }
 
+  const displayValue = statsMs.includes(stat.statHash) ? t('Stats.Milliseconds', { value }) : value;
```

Comment:
so coooool!

Reply:
Neither! Hopefully adding the units will shine a new light on other guardians.